pkg/webhook/validator.go

```diff
@@ -511,18 +522,16 @@ func ValidatePolicySignaturesForAuthority(ctx context.Context, ref name.Referenc
 			sps, err := validSignaturesWithFulcio(ctx, ref, fulcioroot, rekorClient, authority.Keyless.Identities, remoteOpts...)
 			if err != nil {
 				logging.FromContext(ctx).Errorf("failed validSignatures for authority %s with fulcio for %s: %v", name, ref.Name(), err)
-				return nil, fmt.Errorf("validate signatures with fulcio: %w", err)
-			} else if len(sps) > 0 {
-				logging.FromContext(ctx).Debugf("validated signature for %s, got %d signatures", ref.Name(), len(sps))
-				return ociSignatureToPolicySignature(ctx, sps), nil
+				return nil, fmt.Errorf("keyless validation failed for authority %s for %s: %w", name, ref.Name(), err)
 			}
-			logging.FromContext(ctx).Errorf("no validSignatures found for %s", ref.Name())
-			return nil, fmt.Errorf("no valid signatures found with authority %s for  %s", name, ref.Name())
+			logging.FromContext(ctx).Debugf("validated signature for %s, got %d signatures", ref.Name(), len(sps))
+			return ociSignatureToPolicySignature(ctx, sps), nil
 		}
 	}
+
 	// This should never happen because authority has to have been
 	// validated to be either having a Key or Keyless
-	return nil, fmt.Errorf("authority has neither key, keyless, or static specified")
+	return nil, errors.New("authority has neither key, keyless, or static specified")
```

Comment:
while changing this, mind fixing the comment above that I failed to update :)

Reply:
Yup, done!